api/index.js

```diff
@@ -87,9 +87,13 @@ var endpoint = config.api.protocol + '://' +
                config.api.host + ':' + config.api.port +
                '/api/v0/events/birth';
 
-var requestData = function requestData(url, callback) {
+var requestData = function requestData(url, user, callback) {
   return new Promise(function requestDataPromise(resolve, reject) {
-    return requestWithOAuth2.get(url, oauthUrl, clientId, clientSecret, oAuthUsername, oAuthPassword,
+    const headers = user ? {'X-Auth-Downstream-Username': user}: {};
+    return requestWithOAuth2.get({
+      'url': url,
+      'headers': headers
+    }, oauthUrl, clientId, clientSecret, oAuthUsername, oAuthPassword,
```

Comment:
Umm, it'll take either a string or object... You can be lazy and just add a string or be granular.